docs/current_docs/api/daggerverse.mdx

```diff
@@ -0,0 +1,122 @@
+---
+slug: /api/daggerverse
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Daggerverse
+
+Dagger Functions are packaged and shared using Dagger [modules](../features/modules.mdx).
+
+## Ranking
+
+Daggerverse is an index of all known public Dagger modules. To help users find the best quality modules, a basic rank is applied to modules for sorting the module index.
```

Comment:
```suggestion
The Daggerverse is a free service run by Dagger, which indexes all publicly available Dagger Functions, and lets you easily search and consume them. To help users find the best quality modules, a basic rank is applied to modules for sorting the module index.
```

Suggesting to keep consistency with other definitions of the Daggerverse in these docs.